opentracing-spring-cloud-starter-jaeger-it/README.md

```diff
@@ -0,0 +1,9 @@
+## Prerequisites
+
+A running docker daemon needs to be present on the system before running the tests
+
+## Run
+
+```bash
+mvn clean test
```

Comment:
Plese add it to the main repo under development section and no readme here

Reply:
Will do that tomorrow, thanks!